notebooks/2021-08-16/Sun Angles.ipynb

```diff
@@ -0,0 +1,248 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.    # seems to take about 2 seconds per 1000 datapoints. Is this fast enough?
</pre><p>Yeah, 2ms per example should be plenty fast enough :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openclimatefix/nowcasting_dataset/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>